### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by h4ckm03d**
- [x] Manager unit test
- [x] User unit test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
 - *Delivery*
 in this it's delivery protocol, so can be multiple delivery (grpc, rest API, Graphql)
 
 ## How To Run
+
+Copy env sample `cat env.sample > .env`
+
+### Run test
+
+`make test`
```
Comment (hellowin) [inline README.md:29]: NICE!

File: delivery/http/routing/userrouter.go [hunk 2]
```diff
@@ -3,18 +3,19 @@ package routing
...
 	"errors"
+	"net/http"
+
 	"github.com/go-playground/validator/v10"
 	"github.com/gorilla/context"
 	"github.com/gorilla/mux"
-	"kanca-studio/palang/delivery/http/schema"
-	"kanca-studio/palang/manager"
-	"net/http"
+	"github.com/kanca-studio/palang/delivery/http/schema"
```
Comment (hellowin) [inline delivery/http/routing/userrouter.go:11]: What's different between using `github.com` and not? While in `go.mod` we already stated the package name.

**Review by SeptiyanAndika — APPROVED** [review]
Ok

File: delivery/http/routing/userrouter.go [hunk 3]
```diff
@@ -3,18 +3,19 @@ package routing
...
 )
 
 var validate *validator.Validate
-var userManager manager.UserManager
+var userManager manager.User
 
-func UserRouter(router *mux.Router, _userManager manager.UserManager) {
+func UserRouter(router *mux.Router, _userManager manager.User) {
 	validate = validator.New()
 	userManager = _userManager
```
Comment (hellowin) [inline delivery/http/routing/userrouter.go]: Why we need to add `_` as `userManager` prefix and why we re-declare it as `userManager` variable?

**Review by hellowin — COMMENTED** [review]
Only comments

File: delivery/http/routing/userrouter.go (hunk 2 above)
Comment (h4ckm03d) [reply delivery/http/routing/userrouter.go:11]: Got error if run `make test` if not using `github.com`

```bash
make test
go test -v -coverprofile=cover.out ./...
build github.com/kanca-studio/palang/cmd/api: cannot load kanca-studio/palang/database: malformed module path "kanca-studio/palang/database": missing dot in first path element
make: *** [test] Error 1
```

File: delivery/http/routing/userrouter.go (hunk 3 above)
Comment (SeptiyanAndika) [reply delivery/http/routing/userrouter.go]: sorry that's my habit, use prefix `_` in function params, if name same with property/variable class.  will change it without `_`.

Comment (h4ckm03d) [reply delivery/http/routing/userrouter.go]: updated

File: README.md (hunk 1 above)
Comment (h4ckm03d) [reply README.md:29]: Please help integrate this repo using codecov.io (I'm not administrator ), the result looks like this PR https://github.com/bukalapak/envsync/pull/11

**Comment by SeptiyanAndika** [comment]
can we merge this @h4ckm03d @hellowin ?

**Pull request merged**